src/modules/profile/views/waitingTimeStatistics/waitingTimeStatistics.tsx

```diff
@@ -0,0 +1,89 @@
+import { ArcElement, Chart as ChartJS, type ChartOptions, Legend, Tooltip } from 'chart.js';
+import { useEffect, useMemo, useState } from 'react';
+import { Pie } from 'react-chartjs-2';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+interface statistics {
+  waiting_time: number,
+  waiting_time_title: string,
+  waiting_time_count: number,
+  waiting_time_percent: number
+}
+
+interface WaitingTimeStatisticsProps {
+  slug: string;
+  className?: string;
+  statistics: statistics[];
+}
+
+const WaitingTimeStatistics = (props: WaitingTimeStatisticsProps) => {
+  const { className, statistics = [] } = props;
+  const [options, setOptions] = useState<ChartOptions<'pie'>>({});
+
+  const datasets = useMemo(() => {
+    const total = statistics.length
+    return {
+      total: total,
+      data: statistics,
+      values: statistics.map((wt: any) => wt.waiting_time_percent),
+      legendLabels: statistics.map((wt, index) => `%‎${wt.waiting_time_percent.toLocaleString('fa')} ` + wt.waiting_time_title),
```

Comment:
The string concatenation in ``legendLabels: statistics.map((wt, index) => `%‎${wt.waiting_time_percent.toLocaleString('fa')} ` + wt.waiting_time_title)`` can be replaced with template literals for consistency and readability: `` `%‎${wt.waiting_time_percent.toLocaleString('fa')} ${wt.waiting_time_title}` ``.